code/game/objects/items.dm

```diff
@@ -670,7 +670,7 @@ var/list/global/slot_flags_enumeration = list(
 /obj/item/proc/showoff(mob/user)
 	for (var/mob/M in view(user))
 		if(!user.is_invisible_to(M))
-			M.show_message("<b>[user]</b> holds up [src]. <a HREF=?src=\ref[M];lookitem=\ref[src]>Take a closer look.</a>",1)
+			M.show_message("<b>[user]</b> holds up [icon2html(src)] [src]. <a HREF=?src=\ref[M];lookitem=\ref[src]>Take a closer look.</a>",1)
```

Comment:
This is not proper usage of icon2html. Did you test this? It won't work.

Icon2html needs a second argument, the list of people that would receive the icon. Here that would be `viewers(user)` or similiar.